tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "moduleResolution": "node",
     "rootDir": "src",
     "outDir": "lib",
-    "target": "es5"
+    "target": "es6"
```

Comment:
just for consider, could change `lib: ["ES6"]` [at same time](https://www.typescriptlang.org/tsconfig#lib) just to visually drive the two are aligned.

Reply:
good catch and good call! :clap: